src/spider/worker/worker.cpp

```diff
@@ -411,6 +451,18 @@ auto main(int argc, char** argv) -> int {
         return cCmdArgParseErr;
     }
 
+    // Ignore SIGTERM
```

Comment:
This comment is wrong/outdated right? Since now the worker propagates the signal.
```suggestion
```